django_netjsonconfig/static/django-netjsonconfig/js/unsaved_changes.js

```diff
@@ -27,28 +29,29 @@
                     try{
                         object[name] = JSON.parse(value);
                     }
-                    catch(e){}
+                    catch(ignore){}
```

Comment:
you may try as well `catch(){}`

Reply:
jslint dies there if the `ignore` is deleted.
```
 #1 Expected an identifier and instead saw ')'.
    catch(){} // Line 30, Pos 27
 #2 Stopping. (28% scanned).
     // Line 30, Pos 27
```

Reply:
ok